### PR TITLE
Set Endpoint Samples

### PR DESCRIPTION
**Pull request by nnegrey**

### CODE REVIEW

File: automl/beta/pom.xml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    -->
+    <parent>
+        <groupId>com.google.cloud.samples</groupId>
+        <artifactId>shared-configuration</artifactId>
+        <version>1.0.11</version>
+    </parent>
+
+    <properties>
+        <maven.compiler.target>1.8</maven.compiler.target>
+        <maven.compiler.source>1.8</maven.compiler.source>
```
Comment (dzlier-gcp) [inline automl/beta/pom.xml]: Can we use 11?

File: automl/beta/src/main/java/com/example/automl/SetEndpoint.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+import com.google.cloud.automl.v1beta1.ListDatasetsRequest;
+import com.google.cloud.automl.v1beta1.LocationName;
+
+import java.io.IOException;
+
+class SetEndpoint {
+
+  // Change your endpoint
+  static void setEndpoint(String projectId) throws IOException {
+    // [START automl_set_endpoint]
```
Comment (dzlier-gcp) [inline automl/beta/src/main/java/com/example/automl/SetEndpoint.java]: Please have the tag include the imports and the request placement.

File: language/cloud-client/src/main/java/com/example/language/SetEndpoint.java [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+import com.google.cloud.language.v1.LanguageServiceSettings;
+import com.google.cloud.language.v1.Sentiment;
+
+import java.io.IOException;
+
+class SetEndpoint {
+
+  // Change your endpoint
+  static void setEndpoint() throws IOException {
+    // [START language_set_endpoint]
```
Comment (dzlier-gcp) [inline language/cloud-client/src/main/java/com/example/language/SetEndpoint.java]: As above

File: automl/beta/src/main/java/com/example/automl/SetEndpoint.java (hunk 2 above)
Comment (nnegrey) [reply automl/beta/src/main/java/com/example/automl/SetEndpoint.java]: So this tag isn't a full snippet, it is going on a special page where they only want just the code to set the endpoint. 
But I have a full snippet so we can test it.

Comment (dzlier-gcp) [reply automl/beta/src/main/java/com/example/automl/SetEndpoint.java]: OK fair enough

File: automl/beta/pom.xml (hunk 1 above)
Comment (nnegrey) [reply automl/beta/pom.xml]: Done

**Comment by nnegrey** [comment]
Ping

**Comment by nnegrey** [comment]
TODO: Update docs links to master branch.

**Pull request merged**